releasenotes/notes/bump-magnum-capi-helm-6723d89456e6a590.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
```

Comment:
nit: I think upgrade tags are for notes that users should be aware of when upgrading, like a variable name changing or something non-backwards-compatible. This should probably either be treated like a new feature or a fix to an old one

Reply:
Well - here's a description of categories: https://docs.openstack.org/reno/latest/user/usage.html#editing-a-release-note
I think upgrade is a bit better than feature (more people read upgrade section than others) - it might fit other, but then nobody cares

Reply:
I agree with Alex here. Upgrade is for cases that may require user action or be backwards incompatible.

Reply:
done